src/tools/Playwright.Tooling/DriverDownloader.cs

```diff
@@ -113,6 +113,7 @@ private async Task DownloadDriverAsync(DirectoryInfo destinationDirectory, strin
         }
 
         using var client = new HttpClient();
+        client.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
note: this does not affect end-users, this gets only used before publishing and for local development.